src/main/kotlin/lotto/Rank.kt

```diff
@@ -2,11 +2,12 @@ package lotto
 
 import java.math.BigDecimal
 
-enum class Rank(val matchCount: Int, val prize: Amount) {
+enum class Rank(val matchCount: Int, val prize: Amount, val isBonus: Boolean = false) {
     FIRST(6, Amount(BigDecimal(2_000_000_000))),
-    SECOND(5, Amount(BigDecimal(1_500_000))),
-    THIRD(4, Amount(BigDecimal(50_000))),
-    FOURTH(3, Amount(BigDecimal(5_000))),
+    SECOND(5, Amount(BigDecimal(30_000_000)), true),
```

Comment:
```suggestion
    SECOND(5, Amount(BigDecimal(30_000_000)), isBonus = true),
```
다른 개발자들도 어떤 의미인지 한 눈에 이해할 수 있도록 Kotlin의 [named arguments](https://kotlinlang.org/docs/functions.html#named-arguments)를 활용해보시면 어떨까요~?